### DISCUSSION TITLE
Is `form.data` and `form.requestData` behaving correctly for `DateElement`?

### DISCUSSION THREAD

**Discussion by julisch94** [vueform/vueform#128] [Questions] [answered]
Hey there! It's me again 😄 

I have an issue with the `DateElement`. Its user input is not included in `form.data` – only in `form.requestData`.

I've built a minimal reproduction of the issue:

| example form | data does not include date |
| --- | --- |
| <img width="621" alt="image" src="https://github.com/vueform/vueform/assets/22644876/60b4aef6-62de-41d7-b6f6-a9664f4ed315"> | <img width="627" alt="image" src="https://github.com/vueform/vueform/assets/22644876/e54e69c2-592b-423a-9c73-3f8f81613c06"> |

I'm not sure if I'm misunderstanding the docs here. As far as I understand [data](https://vueform.com/reference/vueform#property-data) and [requestData](https://vueform.com/reference/vueform#property-request-data), `data` should also contain the user input from `DateElement`.

From what I'm reading in the docs, I think I'll use `requestData` anyway. I just wanted to understand why `data` wouldn't include my user input.

Here's my component for context:
```html
<template>
  <Vueform ref="bookingForm" :endpoint="false" @submit="onSubmit">
    <template #empty>
      <FormSteps>
        <FormStep
          name="page0"
          :elements="['container']"
          label="Test step"
          :labels="{
            previous: 'Back',
            next: 'Done',
          }"
        />
      </FormSteps>

      <FormElements>
        <GroupElement name="container">
          <DateElement name="event-date" label="Pick a date" />
        </GroupElement>
      </FormElements>

      <FormStepsControls />
    </template>
  </Vueform>
</template>

<script setup lang="ts">
import { ref } from 'vue'
import { type Vueform } from '@vueform/vueform'

const bookingForm = ref<Vueform>()

const onSubmit = () => {
  console.log('bookingForm.value.data', bookingForm.value?.data)
  console.log('bookingForm.value.requestData', bookingForm.value?.requestData)
}
</script>
```

**Comment by adamberecz** [comment, marked as answer]
Hi Julian! It seems like data for `DateElement` is not set if used within a `GroupElement`. I've opened an issue for it, I'll look into it:
https://github.com/vueform/vueform/issues/131

**Reply by adamberecz** [reply]
Fixed
